backend/src/main/resources/graphql/main.graphqls

```diff
@@ -69,10 +70,16 @@ input UpdateDeviceType {
   model: String!
   loincCode: String!
   swabTypes: [ID!]!
-  supportedDiseases: [ID!]!
+  supportedDiseases: [ID!]
+  supportedDiseaseTestPerformed: [SupportedDiseaseTestPerformedInput!]
```

Comment:
It looks like the supported disease requirement dropped off this mutation (you went from `supportedDiseases: [ID!]!` to `supportedDiseases: [ID!]`) - is that intentional?

Reply:
Ahhh I see, thank you!